src/gql/queries/get-patch-task-statuses.ts

```diff
@@ -0,0 +1,11 @@
+import gql from "graphql-tag";
+
+export const GET_PATCH_TASK_STATUSES = gql`
```

Comment:
Instead of creating a new query can we use the already existing Patch query since they both fetch the same info. So apollo knows to fetch the data from the cache. Having a separate query for this data doubles the amount of requests going to the network.

Reply:
they do not fetch any of the same data

Reply:
Sorry i thought they did but just took a look at the patch query and your right.